docs/RELEASE-NOTES.rst

```diff
@@ -7,6 +7,25 @@ Release Notes for BIG-IP Controller for Cloud Foundry
 Added Functionality
 ```````````````````
 
+v1.1.0
+------
+
+Added Functionality
+```````````````````
+* L7 (HTTP) per route configuration through Route Services when controller is run as a Service Broker
```

Comment:
* Use the Controller as a Service Broker to apply per-route L7 (HTTP) configurations